response.proto

```diff
@@ -230,6 +230,10 @@ message Section {
     optional string id = 20;
     optional Co2Emission co2_emission = 21;
     repeated SectionAdditionalInformationType additional_informations = 22;
+    
+    // via which Entrance/Exit
+    optional PtObject via_entrance = 31;
+    optional PtObject via_exit = 32;
```

Comment:
If the entrance/exits are always AccessPoint, let's specify it in the type.
I also suggest to use origin_entrance/destination_exit to be consistent with the origin/destination fields
```suggestion
    optional AccessPoint origin_entrance = 31;
    optional AccessPoint destination_exit = 32;
```

Reply:
It was expected that should be a PtObject... a via can be AccessPoint, it could be something else later